src/store/name-generator/hardcoded-data.ts

```diff
@@ -0,0 +1,150 @@
+export const initialState = {
+  petNames: {
+    boys: [],
+    girls: [],
+  },
+  girlId: 0,
+  boyId: 0,
+  errors: '',
+  loading: false,
+};
```

Comment:
nit: As far I know you can export "real" (existing) initialState or create it from the scratch following the state schema. I mean, you don't need to create untyped object in the different file.

Treat this comment like general information, not a recommendation :)

Reply:
indeed, corrected)